### DISCUSSION TITLE
Unable to access server

### DISCUSSION THREAD

**Discussion by ghost** [saran-surya/email_auth#60] [Q&A]
Hi every one, I'm using the default server of this package, but today I got error "Unable to access server" and cannot send otp anymore, anyone helps me out with that? Thanks in advance.

**Comment by saran-surya** [comment]
Hi @duyhuynh-arc97 please refer this issue for a better understanding, Also please set up a production server to have a reliable connection.

https://github.com/saran-surya/email_auth/issues/58#issuecomment-1003492968
